cs/config.go

```diff
@@ -7,17 +7,19 @@ import (
 	"net/url"
 	"os"
 	"path/filepath"
+	"strings"
 
 	"github.com/AlecAivazis/survey/v2"
 	"github.com/fatih/color"
-	"github.com/google/go-github/v47/github"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
 var defaultCfgFile = ".codesearch"
 var token string
 
+const defaultBaseURL string = "https://api.github.com/"
```

Comment:
Will think about it as I read the rest, but this feels like a good candidate for `viper.SetDefault()` (see L40). I want us to make it hard to have configs in a bad state.